drivers/driver.go

```diff
@@ -144,8 +165,10 @@ func (r RDMAVolumeDriver) Remove(request volume.Request) volume.Response {
 	// Ensure the r is properly confiured
 	r.validateOrCrash()
 
-	// Pass the remove request to the storage controller.
-	err := r.StorageController.Remove(request.Name)
+	// TODO unmount all mounts before remove
```

Comment:
TODOs should be made into issues and linked to the issue in the code comment with at least a #number. Otherwise, they should be removed.